apps/prairielearn/src/middlewares/selectOpenIssueCount.ts

```diff
@@ -5,12 +5,16 @@ import * as sqldb from '@prairielearn/postgres';
 
 const sql = sqldb.loadSqlEquiv(import.meta.url);
 
+export interface ResLocalsCourseIssueCount {
+  navbarOpenIssueCount: number;
+}
+
 export default asyncHandler(async (req, res, next) => {
-  const result = await sqldb.queryRow(
+  const count = await sqldb.queryRow(
     sql.select_open_issue_count,
     { course_id: res.locals.course.id },
     z.number(),
   );
-  res.locals.navbarOpenIssueCount = result;
+  res.locals.navbarOpenIssueCount = count;
```

Comment:
Nit, may as well combine them:

```suggestion
  res.locals.navbarOpenIssueCount = await sqldb.queryRow(
    sql.select_open_issue_count,
    { course_id: res.locals.course.id },
    z.number(),
  );
```